src/pages/ExcelOpplasting.less

```diff
@@ -78,13 +68,29 @@
   .filKnapp {
     margin: 1rem 0rem;
   }
+
+  .filKnapp[disabled] {
+    pointer-events:none
```

Comment:
Denne har jeg lagt til for å catche når noen prøver å trykke på den disabla  submit-knappen, da det bare skal vises feilinfo da. Er det innafor?